website/docs/getting-started/iOS-all-in-one.md

```diff
@@ -0,0 +1,216 @@
+---
+title: iOS Quick Start
+sidebar_label: iOS Quick Start
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import CodeBlock from '@theme/CodeBlock';
+import { getCapacitorVersion, getPortalsVersion, getPortalsVersionIos, getPortalsVersionAndroid, getPortalsVersionRN, getiOSMinVersion, getAndroidMinSdk, getRnMinVersion } from '@site/src/util';
+
+# iOS Quick Start
+
+This is a walkthrough on how to get a single Portal + web application setup. To begin this walkthrough it will be assumed that you have already signed up for access to Portals and you have your Portals key ready.
+
+First have your web application ready. We will add some configuration to it and then get it setup in Appflow.
+
+## 1. Create a capacitor config
+
+Create a `capacitor.config.json` file to the root of your web project. We will not need to add Capacitor to the web project but this configuration file will help us configure some options for the Portals that is displaying this web application.
```

Comment:
fix grammar ' ...options for the Portal that **are** displaying...'

Reply:
Fixed